test-outofproc/AddProductColumnTypes.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Azure.Functions.Worker.Http;
+using System.Web;
+using System.Collections.Specialized;
+using Microsoft.Azure.Functions.Worker.Extension.Sql;
+using DotnetIsolatedTests.Common;
+using System;
+
+namespace DotnetIsolatedTests
+{
+    public static class AddProductColumnTypes
+    {
+        /// <summary>
+        /// This function is used to test compatability with converting various data types to their respective
+        /// SQL server types. 
+        /// </summary>
+        [Function(nameof(AddProductColumnTypes))]
+        [SqlOutput("dbo.ProductsColumnTypes", ConnectionStringSetting = "SqlConnectionString")]
+        public static ProductColumnTypes Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "addproduct-columntypes")] HttpRequestData req)
+        {
+            NameValueCollection queryStrings = HttpUtility.ParseQueryString(req.Url.Query);
+            var product = new ProductColumnTypes()
+            {
+                ProductID = int.Parse(queryStrings["productId"], null),
+                Datetime = DateTime.UtcNow.Date,
```

Comment:
Does the test pass using DateTime.UtcNow.Date?

Reply:
It doesn't on my local machine, it still throws the conversion error but forgot to revert back that change and it passed on the pipeline!

Reply:
Excluded what? The test seems to be running against OOP here from what I can tell

Reply:
Oh I just added it back to check if it's passing on the pipeline after I commented here.